sdk/typescript/src/builder/export.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export { Inputs } from './Inputs.js';
+export { Transactions } from './Transactions.js';
+export { TransactionBlock } from './TransactionBlock.js';
+export { TransactionBlockDataBuilder } from './TransactionBlockData.js';
```

Comment:
I was looking through what we currently export, and I'm not sure we need much else here.  Would love a second opinion though.

This is a rough outline of existing exports:

- WellKnownEncoding (internal)
- TRANSACTION_TYPE (seems like it should be internal)
- Transactions (exported)
- Inputs (exported)
- TransactionBlockDataBuilder (exported)
- SerializedTransactionDataBuilder (internal I think)
- TransactionBlock (exported)
- isTxContext (internal?)
- getPureSerializationType (internal?)
- PureCallArg (can be created via Input.Pure(...))
- ObjectCallArg
- PureCallArg
- ObjectCallArg
- BuilderCallArg
- getIdFromCallArg (internal?)
- getSharedObjectInput (internal?)
- isSharedObjectInput (internal?)
- isMutableSharedObjectInput (internal?)
- bcs type names (internal?)
- builder (bcs builder, not sure if this should be internal, or exported with a different name)

Reply:
Do we need to export the data builder?

Reply:
not sure, maybe not